tokens/pacha.svg

```diff
@@ -0,0 +1 @@
.456 27.397,317.896 99.723 C 357.178 145.223,363.764 210.781,335.436 274.300 C 319.516 309.997,297.175 336.282,257.000 366.585 C 247.859 373.480,248.907 376.268,259.800 374.036 C 275.179 370.884,298.204 358.830,314.200 345.556 C 353.968 312.556,378.123 268.175,384.167 217.000 C 396.131 115.712,321.374 25.937,217.800 17.213 C 213.452 16.846,192.715 16.729,189.800 17.054 M213.292 19.209 C 319.097 25.076,396.088 117.259,380.823 219.800 C 377.258 243.749,369.077 267.774,358.143 286.400 C 357.562 287.390,356.762 288.773,356.365 289.473 C 355.969 290.173,355.004 291.748,354.222 292.973 C 353.440 294.198,352.147 296.235,351.349 297.500 C 350.552 298.765,349.236 300.700,348.427 301.800 C 347.617 302.900,346.158 304.880,345.185 306.200 C 334.380 320.861,319.039 335.969,306.400 344.394 C 304.310 345.787,302.133 347.243,301.563 347.629 C 299.610 348.950,290.813 353.200,290.030 353.200 C 289.599 353.200,289.136 353.380,289.000 353.600 C 288.355 354.643,286.000 354.041,286.000 352.833 C 286.000 352.192,286.141 351.607,286.312 351.533 C 286.484 351.460,286.902 350.817,287.241 350.105 C 287.580 349.393,288.924 347.674,290.229 346.286 C 291.533 344.897,293.767 342.420,295.192 340.781 C 296.618 339.141,297.885 337.710,298.007 337.600 C 299.349 336.392,309.935 323.946,312.764 320.252 C 313.964 318.683,315.782 316.320,316.803 315.000 C 319.027 312.124,326.767 300.526,329.083 296.600 C 332.193 291.327,332.807 290.188,336.521 282.800 C 342.872 270.164,346.696 260.387,350.522 247.000 C 360.737 211.268,359.766 168.969,348.084 140.800 C 342.223 126.666,339.362 121.196,332.357 110.731 C 324.494 98.985,312.466 85.835,301.799 77.324 C 300.919 76.623,299.210 75.255,298.000 74.285 C 295.848 72.560,282.649 63.600,282.260 63.600 C 282.152 63.600,280.564 62.737,278.732 61.682 C 229.843 33.542,161.844 36.070,116.723 67.706 C 115.394 68.638,113.498 69.970,112.511 70.666 C 94.427 83.418,77.980 103.969,67.857 126.460 C 56.561 151.561,52.499 187.260,58.009 213.000 C 73.704 286.306,152.854 325.005,219.225 291.825 C 273.609 264.636,297.824 198.290,270.038 152.600 C 261.663 138.829,245.623 125.463,231.148 120.193 C 230.186 119.843,228.410 119.199,227.200 118.762 C 183.608 103.022,136.549 129.689,127.958 175.000 C 120.656 213.519,150.042 239.925,183.400 224.819 C 202.002 216.395,207.567 197.537,195.661 183.266 C 185.113 170.623,198.669 160.691,215.400 168.804 C 231.027 176.383,235.848 194.772,227.551 215.156 C 205.812 268.564,129.443 265.924,107.315 211.000 C 101.160 195.723,101.100 171.044,107.177 154.400 C 122.122 113.473,155.041 90.601,199.000 90.601 C 267.386 90.601,317.674 148.091,306.766 213.800 C 292.567 299.325,205.430 349.662,124.000 319.378 C 46.019 290.376,13.455 204.396,47.505 117.400 C 49.177 113.129,54.342 102.468,56.420 99.000 C 57.079 97.900,57.736 96.730,57.881 96.400 C 58.377 95.264,64.517 86.193,67.221 82.600 C 97.271 42.677,145.778 19.944,203.200 18.871 C 204.850 18.841,209.391 18.992,213.292 19.209 " stroke="none" fill="#2fd0e1" fill-rule="evenodd"></path></g></svg>
```

Comment:
The icon is too big. Make sure it's 5-7 KB max.

Reply:
I sent uploaded a new one and did a pull requested and merged.